File: unix/boot/spp/rpp/ratlibf/mktabl.f
      integer function mktabl (nodsiz)
      integer nodsiz
      integer mem( 60000)
      common/cdsmem/mem
      integer st
      integer dsget
      integer i
      st = dsget (43 + 1)
      mem (st) = nodsiz
      mktabl = st
      do 23000 i = 1, 43
      st = st + 1
      mem (st) = 0
23000 continue
23001 continue
      return
      end

File: unix/boot/spp/rpp/ratlibr/mktabl.r
include	defs

# mktabl --- make a new (empty) symbol table

   pointer function mktabl (nodsiz)
   integer nodsiz

   DS_DECL(Mem, MEMSIZE)

   pointer st
   pointer dsget

   integer i

   st = dsget (ST_HTABSIZE + 1)     # +1 for record of nodsiz
   Mem (st) = nodsiz
   mktabl = st
   do i = 1, ST_HTABSIZE; {
      st = st + 1
      Mem (st) = LAMBDA 	    # null link
      }

   return
   end
